modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -483,21 +485,17 @@ public Collection<Entry> getAndPutAll(List<byte[]> keys, List<byte[]> values) {
         return res;
     }
 
-    /** {@inheritDoc} */
-    @NotNull
     @Override
     public Entry get(byte[] key) {
         rwLock.readLock().lock();
```

Comment:
This part was not written by me, but I believe that these locks are needed to prevent removing of data during Meta Storage compaction (since we remove data from two Column Families separately). Compaction is not used anywhere yet, but I think this stuff should be re-worked altogether when we will implement the GC mechanism.

Reply:
Ok, thanks for the explanation